plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/cache/DeltaLakeCacheKeyProvider.java

```diff
@@ -13,24 +13,42 @@
  */
 package io.trino.plugin.deltalake.cache;
 
+import com.google.inject.Inject;
 import io.trino.filesystem.TrinoInputFile;
 import io.trino.filesystem.cache.CacheKeyProvider;
+import io.trino.plugin.deltalake.DeltaLakeConfig;
 
 import java.util.Optional;
 
 import static io.trino.plugin.deltalake.statistics.MetaDirStatisticsAccess.STARBURST_META_DIR;
 import static io.trino.plugin.deltalake.statistics.MetaDirStatisticsAccess.STATISTICS_META_DIR;
+import static io.trino.plugin.deltalake.transactionlog.TransactionLogUtil.TRANSACTION_LOG_DIRECTORY;
 
 public class DeltaLakeCacheKeyProvider
         implements CacheKeyProvider
 {
+    private final boolean deltaLogFileSystemCacheDisabled;
```

Comment:
nit: deltaLogCacheDisabled - `FileSystem` looks a bit redundant to me

Reply:
Done!

Reply:
@Praveen2112 was that comment meant only for this class or DeltaLakeConfig in general ?
We might want to retain `FileSystem` name in the config because there are other in-memory delta caches which this might get confused with.

Reply:
@raunaqmorarka I meant it only here in the `CacheKeyProvider` - I guess `CacheKeyprovider` is used only for FileSystem so it could be redundant. In case of Config - we could maintain it so `filesystem` as a part of the config

Reply:
Ok renamed `deltaLogFileSystemCacheDisabled` -> `deltaLogCacheDisabled` only in `DeltaLakeCacheKeyProvider`